### ISSUE TITLE
DOOM Compatibility settings bug FIGURED OUT [update to #1439]

### ISSUE THREAD

**Issue by snekyo** [fabiangreffrath/woof#1459]
So three days ago, I made an entry in the issues list about how the DOOM Compatibility settings were bugged (#1439), only showing up sometimes.

Well, I found out why. I was using the vanilla complevels for authenticity reasons, not knowing that the vanilla complevels actually locked the DOOM Compatibility settings due to their limitations and lack of compatibility with them. Setting them to MBF21 makes them selectable.

Despite this issue being resolved, I think it still would be good to implement the feature fabiangreffrath thought up **(seen below)** about stopping any playing DEMO when the general or setup menu is opened.

> This is a good idea! We should stop all demos from the reel when entering a Setup menu and prevent any other demo to start.

I'm submitting this to make sure that the developers of Woof know that the bug has been resolved :)

-snekyo

**Comment by rfomin** [comment]
> Despite this issue being resolved, I think it still would be good to implement the feature fabiangreffrath thought up **(seen below)** about stopping any playing DEMO when the general or setup menu is opened.

We have redesigned the Doom Compatibility menu in the development version. Try the latest artifact from this page: https://github.com/fabiangreffrath/woof/actions/runs/7853454120
![image](https://github.com/fabiangreffrath/woof/assets/5077629/d41f85d7-1b2d-4e96-86fa-e7136e444015)

**Comment by snekyo** [comment]
I appreciate the offer, but I use Nugget DOOM. I use to use Woof, which is when I noticed this bug. I told the DOOM compatibility issue to the Nugget DOOM developer, which is when I figured out that I should be using MBF21 instead of Vanilla to change the comp settings. I decided to go back to the Woof page and let you guys know the issue as Nugget is a fork of Woof and this one I also thought was bugged because of this reason.
